src/chainstate/stacks/db/blocks.rs

```diff
@@ -3973,6 +4024,11 @@ impl StacksChainState {
             let mut receipts =
                 StacksChainState::process_stacking_ops(&mut clarity_tx, stacking_burn_ops);
 
+            receipts.extend(StacksChainState::process_transfer_ops(
```

Comment:
Should we process on-Bitcoin operations in the order in which they appear on-Bitcoin?  I don't think correctness is a problem either way, but it may be surprising to users that on-Bitcoin operations aren't processed in the same order as they appear in the block (all Stacking operations happen in block-order, and then all STX-transfer operations happen in block-order).

Reply:
Yep, switched to an ordered processing in https://github.com/blockstack/stacks-blockchain/pull/2129/commits/3320c16534e986ae33a118a7964db39722df7ff0